config/initializers/secret_token.rb

```diff
@@ -4,4 +4,4 @@
 # If you change this key, all old signed cookies will become invalid!
 # Make sure the secret is at least 30 characters and all random,
 # no regular words or you'll be exposed to dictionary attacks.
-CampusCodefest::Application.config.secret_token = '575ca50571c5ba792cac840e79f7b8b8362d420f7c56cdd8ae3040c994d9e459431e3f33f0138bc68e3b0e54ee1de9f9d4448e6b065208fed2ae2eb7f28ade4a'
+CampusCodefest::Application.config.secret_token = ENV['CCF_SECRET_TOKEN']
```

Comment:
Oh, right. I tend not to use ENV vars over config files these days and totally forgot that we are on a yml file config situation. I switched because it made CI easier. I have also been meaning to switch over to https://github.com/bkeepers/dotenv to handle local dev to get kind of the best of both worlds. 

I suppose this particular item might fit better into our config file for now. 

BTW, do you have a preference (ENV vs config file)?